discovery-provider/src/app.py

```diff
@@ -583,6 +589,7 @@ def configure_celery(celery, test_config=None):
     redis_inst.delete("calculate_trending_challenges_lock")
     redis_inst.delete("index_user_listening_history_lock")
     redis_inst.delete("prune_plays_lock")
+    redis_inst.delete(INDEX_REACTIONS_LOCK)
```

Comment:
Oh oops, I probably need this for my `index_aggregate_users`?

Reply:
yeah, we should figure out how to do this programmatically...